native/vendor/google/play/TResult.h

```diff
@@ -122,28 +122,40 @@ class Achievement {
 class AchievementBuffer {
 public:
     ~AchievementBuffer() {
-        for(auto* achievement : _achievements) {
-            delete achievement;
-        }
-        _achievements.clear();
+        release();
+    }
+
+    Achievement* createAchievement() {
+        auto* achievement = new Achievement(); // --> new elements also in AchievementBuffer class

```

Comment:
Remove the comment which in my reply. It just a description of why it should be moved to AchievementBuffer class.

Reply:
fixed